autogluon/utils/tabular/ml/trainer/abstract_trainer.py

```diff
@@ -983,27 +984,60 @@ def save_model(self, model, reduce_memory=True):
             self.models[model.name] = model
 
     def save(self):
+        models = self.models
+        if self.low_memory:
+            self.models = {}
         save_pkl.save(path=self.path + self.trainer_file_name, object=self)
+        if self.low_memory:
+            self.models = models
 
-    def load_models_into_memory(self, model_names=None):
+    def persist_models(self, model_names: list = None, with_ancestors=False, max_memory=None) -> list:
         if model_names is None:
             model_names = self.get_model_names_all()
+        elif model_names is 'best':
+            if self.model_best is not None:
+                model_names = [self.model_best]
+            else:
+                model_names = [self.get_model_best(can_infer=True)]
+        if with_ancestors:
```

Comment:
Should there be a check that `model_names` is valid argument before this moving on with this line?

Reply:
added